packages/tinacms/src/styles.css

```diff
@@ -1,16 +1,16 @@
-.tina-tailwind {
-  @tailwind base;
-  @tailwind components;
-  @tailwind utilities;
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
This will be affected by the `important: '.tina-tailwind'` declaration in the config.